fdbbackup/backup.actor.cpp

```diff
@@ -1395,6 +1622,13 @@ ProgramExe getProgramType(std::string programExe) {
 		enProgramExe = ProgramExe::DB_BACKUP;
 	}
 
+	// Check if data movement between multi-tenant clusters
```

Comment:
Technically we also will support movement between other clusters

```suggestion
	// Check if moving data between clusters
```